Dockerfile

```diff
@@ -62,9 +62,9 @@ RUN --mount=type=bind,source=./superset-frontend/package.json,target=./package.j
     --mount=type=cache,target=/root/.cache \
     --mount=type=cache,target=/root/.npm \
     if [ "$DEV_MODE" = "false" ]; then \
-        npm ci; \
+    npm ci; \
```

Comment:
why removing the indentation here?

Reply:
Sorry, this wasn't intentional. My IDE seems hell bent on doing this. I'll fix it.